ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -446,13 +456,19 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{app
                                                             />
                                                         </a>
                                                         {(pref.view === 'tree' || pref.view === 'network') && (
-                                                            <a
-                                                                className={`group-nodes-button group-nodes-button${!pref.groupNodes ? '' : '-on'}`}
-                                                                title={pref.view === 'tree' ? 'Group Nodes' : 'Collapse Pods'}
-                                                                onClick={() => this.toggleCompactView(pref)}>
-                                                                <i className={classNames('fa fa-object-group fa-fw')} />
-                                                            </a>
+                                                            <Tooltip
+                                                                content={AppUtils.userMsgsList[showToolTip?.msgKey] || ''}
+                                                                visible={pref.groupNodes && showToolTip && !showToolTip?.display}
```

Comment:
One nit-pick is that the display attribute here:  !showToolTip?.display

Should we initialize the boolean to true first to make this visible, and then set it false to hide it?  Seems more meaningful that way.

Reply:
This is intensional, i want to show the tooltip when it never displayed before